x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_actions/route.ts

```diff
@@ -71,6 +71,13 @@ type BulkActionError =
   | PromisePoolError<RuleAlertType>
   | BulkOperationError;
 
+interface ErrorsAndSkippedRules {
+  skipped: Array<
+    Pick<RuleAlertType, 'name' | 'id'> & { bulk_edit_skip_reason: BulkEditSkipReason }
+  >;
```

Comment:
Suggested in a separate comment what should be the type of the `skipped` property.